src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,23 @@
+:root {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
it's normalize styles, move it to normalize

Reply:
done